webpack/dev.config.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable */
+const webpack = require('webpack');
+var webpackMerge = require('webpack-merge');
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
+var commonConfig = require('./common.config.js');
+const IsomorphicPlugin = require('webpack-isomorphic-tools/plugin');
+const path = require('path')
+const webpackIsomorphicToolsPlugin = new IsomorphicPlugin(require('./isomorphic.config'));
+
+module.exports = webpackMerge(commonConfig, {
+  devtool: 'cheap-module-eval-source-map',
+  output: {
+    path: path.resolve('./build'),
+    filename: '[name].js',
+    chunkFilename: '[id].chunk.js'
+  },
+
+  plugins: [
+    new ExtractTextPlugin('[name].css'),
+    new webpack.ProvidePlugin({
+      $: 'jquery',
```

Comment:
@andela-kadeniyi . Yea it's best to avoid it though as most of this utility functions are already provided in a lighter library like lodash/ramda some are even in es6. so most times it's better to avoid it's import. + $.ajax have problems making request from a server side app